pymc_marketing/mmm/transformers.py

```diff
@@ -715,3 +715,39 @@ def tanh_saturation_baselined(
     Developed by Max Kochurov and Aziz Al-Maeeni doing innovative work in `PyMC Labs <pymc-labs.com>`_.
     """
     return gain * x0 * pt.tanh(x * pt.arctanh(r) / x0) / r
+
+
+def michaelis_menten(
+    x: Union[float, np.ndarray, npt.NDArray[np.float64]],
+    alpha: Union[float, np.ndarray, npt.NDArray[np.float64]],
+    lam: Union[float, np.ndarray, npt.NDArray[np.float64]],
+) -> Union[float, Any]:
+    r"""
+    Evaluate the Michaelis-Menten function for given values of x, alpha, and lambda.
+
+    The Michaelis-Menten function models enzyme kinetics and describes how the rate of a chemical reaction increases with substrate concentration until it reaches its maximum value.
+
```

Comment:
Adding a plot like with the other saturation functions would be awesome!
Example: https://www.pymc-marketing.io/en/latest/api/generated/pymc_marketing.mmm.transformers.tanh_saturation.html#pymc_marketing.mmm.transformers.tanh_saturation

Reply:
Indeed! +1